docs/examples/PanelListGroupFill.js

```diff
@@ -0,0 +1,13 @@
+var panelInstance = (
+    <Panel header="Panel heading">
+      Some default panel content here.
+      <ListGroup fill>
+        <ListGroupItem>Item 1</ListGroupItem>
+        <ListGroupItem>Item 2</ListGroupItem>
+        <ListGroupItem>&hellip;</ListGroupItem>
+      </ListGroup>
+      Some more panel content here.
+    </Panel>
+  );
+
+React.render(panelInstance, mountNode);
```

Comment:
I spot checked on Safari and Chrome that it looks like this on my side:

With `fill`:
![image](https://cloud.githubusercontent.com/assets/3112159/5883368/996d9470-a31f-11e4-85c2-94af16e5dc39.png)

Without `fill`:
![image](https://cloud.githubusercontent.com/assets/3112159/5883358/8b7f466a-a31f-11e4-840a-4ade805a346c.png)